core/src/Ledger.sol

```diff
@@ -165,7 +165,7 @@ contract Ledger {
 
         (Cache memory cache, , ) = _previewInterest(_getCache());
         unchecked {
-            return (b - 1).mulDivUp(cache.borrowIndex, BORROWS_SCALER);
+            return ((b - 1) * cache.borrowIndex) / BORROWS_SCALER;
```

Comment:
See #55

Reply:
These actually aren't equivalent! Previously we were rounding up, now we're rounding down.

(`mulDivUp` and `mulDivDown` are different from `FullMath.mulDiv`. I know it's confusing)